### DISCUSSION TITLE
How Can I completely block Internet to a device from Pi Hole

### DISCUSSION THREAD

**Discussion by aadiveer** [pi-hole/pi-hole#4684] [Bugs]
How Can I completely block Internet from Pi Hole via DNS to a specific device

**Comment by yubiuser** [comment]
Because of the number of support requests and the very limited resources we have as a free open-source project run by volunteers, we ask that all configuration questions be addressed at our Discourse forum (https://discourse.pi-hole.net/)

Please use the search function over there, this question has been answered a few times.

Thank you for your understanding.
